smart_home/WeatherStation.py

```diff
@@ -106,6 +106,7 @@ def moduleById(self, mid, sid=None):
     def monitoredConditions(self, module):
         mod = self.moduleByName(module)
         conditions = []
+        conditions.extend("reachable")
```

Comment:
This line will return conditions =[r,e,a,...]. You should do conditions.extend([”reachable”]) or conditions.append(”reachable”)

Reply:
Yeah, good catch. Although this specific line should actually be deleted. I missed this when excluding support for HomeCoach. This same error was fixed lower down though.